CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the "azure-cosmosdb" extension will be documented in this file.
 
+## 0.10.3 - 2019-09-03
+### Fixed
+- Better shell error handling, add mongo.shell.args setting [#1101](https://github.com/Microsoft/vscode-cosmosdb/issues1101), [#1092](https://github.com/Microsoft/vscode-cosmosdb/issues1092), [#1071](https://github.com/Microsoft/vscode-cosmosdb/issues1071), [#998](https://github.com/Microsoft/vscode-cosmosdb/issues998), [#83](https://github.com/Microsoft/vscode-cosmosdb/issues83), [#820](https://github.com/Microsoft/vscode-cosmosdb/issues820), [#1126](https://github.com/Microsoft/vscode-cosmosdb/issues1126), [#1104](https://github.com/Microsoft/vscode-cosmosdb/issues1104)
```

Comment:
shell error handling is a fix, args should be an "added"

Reply:
These two bugs are for mongo.shell.args: 1104, 1126
This not related to shell fixes: 1115
This not related to shell fixes: 1021
Don't need a link for (not regression from previous release): 1176, 1149